beacon-chain/forkchoice/protoarray/optimistic_sync.go

```diff
@@ -188,17 +187,16 @@ func (f *ForkChoice) UpdateSyncedTips(ctx context.Context, root [32]byte) error
 	return nil
 }
 
-// This function removes a node that has become INVALID according to the
-// EL. It updates the synced_tips to reflect this change.
-func (f *ForkChoice) Invalid(ctx context.Context, root [32]byte) error {
+// RemoveSyncTip removes a node with root from Fork Choice store. It updates the synced tips map.
+func (f *ForkChoice) RemoveSyncTip(ctx context.Context, root [32]byte) error {
```

Comment:
This is a bit misleading since this function may not remove any synced tip, take for example 

```
0  -- 1 
 \ 
   --  2
```
with 0 being the unique synced tip and 1 and 2 optimistic. If 1 becomes invalid, it is removed, but synced tips do not change

Reply:
I'll think of better names for them. I mainly want to express some form of symmetric for the two functions